Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -477,6 +477,56 @@ void TextureCacheBase::BindTextures()
   }
 }
 
+struct ManualMipmapDetector
+{
+
+  ManualMipmapDetector()
+  {
+    min = { 255, 255, 255, 255 };
+    max = { 0, 0, 0, 0 };
+  }
+
+  void AddLevel(u32 width, u32 height, u32 row_length, const u8* buffer)
+  {
+    ++levels;
+
+    u64 avg[4] = { 0 };
+    for (u32 i = 0; i < height; ++i)
+    {
+      auto* row = buffer;
```

Comment:
`const auto*`, should be used to be explicit, even if auto would otherwise apply the const implicitly.